justfile

```diff
@@ -8,6 +8,9 @@ celestia-up:
   echo "Cleaning up any existing Docker resources..."
   docker-compose -f {{DOCKER_COMPOSE_FILE}} down -v --remove-orphans
 
+  echo "Building Docker images..."
+  docker-compose -f {{DOCKER_COMPOSE_FILE}} build
+
```

Comment:
When changing something in the entrypoint, we want the images to be rebuilt before restarting.